components/count-badge/test/count-badge.visual-diff.js

```diff
@@ -14,6 +15,18 @@ describe('d2l-count-badge', () => {
 		await page.bringToFront();
 	});
 
+	async function getRect(page, selector) {
+		return page.$eval(selector, (elem) => {
```

Comment:
The tooltip wasn't getting into the rect clip for some reason, so I just added my own getRect function for the tooltip test.